packages/authentication/package.json

```diff
@@ -19,10 +19,10 @@
   "license": "MIT",
   "dependencies": {
     "@loopback/core": "^4.0.0-alpha.5",
+    "@loopback/context": "4.0.0-alpha.6",
```

Comment:
Please use carrot operator when specifying dependency versions. That way we don't have to change this line whenever a new context version is released.

```
"@loopback/context": "^4.0.0-alpha.6"
````